<article class="bugdoc">
    <h2 class="bugdoc-title" id="Bug-Doc:PE22331">PE22331: B306</h2>
    <h3>B306</h3>
    <section id="tip">
     <h4 style="border-top: 0pt; padding: 0pt 4pt 4pt 4pt;">Bug info</h4>
                <dl>
                    <dt>BugTypeID</dt>
                    <dd>PE22331</dd>
                    <dt>Tags</dt>
                    <dd>flake8</dd>
                    <dt>Category</dt>
                    <dd>Code style and efficiency</dd>
                    <dt>Authors</dt>
                    <dd>pinpoint</dd>
                </dl>
    <h2>Reason</h2>

<pre><code>B306 `BaseException.message` has been deprecated as of Python 2.6 and is removed in Python 3. Use `str(e)` to access the user-readable message. Use `e.args` to access arguments passed to the exception.

</code></pre>
<h4>References</h4>
<section class="footnotes">
<ol>
<li id="fn-1"><p>https://www.python.org/dev/peps/pep-0008/<a href="#fnref-1" class="footnote">&#8617;</a></p></li>
</ol>
</section>

    </section>
</article>
